### ISSUE TITLE
Unhandled Runtime Error

### ISSUE THREAD

**Issue by AugustBemsen** [eltasolutions/react-dojah#3]
I'm getting `NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node` error. This happened when I closed the modal and tried doing something else on my app
![image](https://user-images.githubusercontent.com/54921134/207667352-9031034e-29d7-4aff-b949-7a329c560dda.png)

**Comment by Keith-Web3** [comment]
Hey @AugustBemsen , were you able to resolve this? Currently having the same issue, and I can't seem to fix it

**Comment by AugustBemsen** [comment]
@Keith-Web3 yeah had to render the particular component on client side instead of server. Which is NextJs default 

https://dev.to/danireptor/understanding-and-implementing-nossr-in-nextjs-a-comprehensive-guide-3idm

**Comment by Keith-Web3** [comment]
@AugustBemsen already doing that. However, I'm using the app router.
I eventually just found a workaround by redirecting with window.location anytime the widget closes.
